dashboard/dashboard.js

```diff
@@ -127,7 +127,7 @@ function renderCharts (error, apiData) {
       mapG = mapSVG.append("g").attr("class", "leaflet-zoom-hide");
 
   d3_queue.queue()
-      .defer(d3.json, "../data/j2j3-acqj.json")  /* https://data.sfgov.org/resource/j2j3-acqj.json?$limit=2000 */
+      .defer(d3.json, "../data/j2j3-acqj.json") //this data has already loaded at this point, why is it doing it agian? /* https://data.sfgov.org/resource/j2j3-acqj.json?$limit=2000 */
```

Comment:
I looked into this. Not sure but seems to go back some ways, at least to September 2016